docs/data/charts/lines/InterpolationDemoNoSnap.js

```diff
@@ -15,6 +15,8 @@ const curveTypes = [
   'step',
   'stepBefore',
   'stepAfter',
+  'bumpY',
+  'bumpX',
 ];
```

Comment:
I would be in favor of not documenting those options. I can see the interest for drawing a funnel chart but not a line chart. So I would not bother reader with additional option they might never use